views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -286,16 +297,49 @@ define([
                     if (attrValue === true) {
                         interaction.attr('loop', false);
                         interaction.attr('maxPlays', 0);
+                    } else {
+                        interaction.removeAttr('data-sequence-repeats');
+                        interaction.removeAttr('data-sequence-delay-between-ms');
+                        interaction.removeAttr('data-sequence-delay-after-ms');
                     }
                     renderForm();
                 },
 
+                sequenceRepeats: function repeats(boundInteraction, attrValue) {
+                    let attrValueNumeric = parseInt(attrValue, 10);
+                    attrValueNumeric = Number.isNaN(attrValueNumeric) || attrValueNumeric < 1 ? 1 : attrValueNumeric;
+                    interaction.attr('data-sequence-repeats', attrValueNumeric);
+
+                    if (attrValueNumeric === 1) {
+                        interaction.attr('data-sequence-delay-between-ms', 0);
+                    }
+                    if (interaction.attr('maxPlays') > 0) {
+                        const maxPlays = parseInt($form.find('[name="maxPlays"]').val(), 10);
+                        interaction.attr('maxPlays', maxPlays * attrValueNumeric);
+                    }
```

Comment:
I'm not sure about altering `maxPlays` based on `sequenceRepeats`. I think it's safer to keep the concepts separated, and unset one whenever the other is used.

Reply:
Ok, I'll disable `maxPlays` when repeats are used, it's simpler.

Though logically, I think they are different: `maxPlays` is about how much times can the whole sequence run. `repeats` is about auto-starting playing during the same sequence run. But if customers will ever need that use case, we'll enable it.